### ISSUE TITLE
versions showing incorrect info for macOS

### ISSUE THREAD

**Issue by OmgImAlexis** [sebhildebrandt/systeminformation#172]
### Issue Type

* [x] Bug
* [ ] Feature Request
* [ ] Question

### Expected behavior 

```js
{ kernel: '16.7.0',
  openssl: '1.0.2o',
  node: '11.0.0',
  v8: '7.0.276.28-node.5',
  npm: '6.4.1',
  yarn: '1.12.1',
  pm2: '2.7.2',
  gulp: '',
  grunt: '',
  git: '2.17.1',
  tsc: '2.9.2',
  mysql: '',
  redis: '4.0.10',
  mongodb: '',
  nginx: '',
  php: '5.6.30' }
```

### Actual behavior

```js
{ kernel: '16.7.0',
  openssl: '1.1.0i \n',
  node: '11.0.0',
  v8: '7.0.276.28-node.5',
  npm: '6.4.1',
  yarn: '1.12.1',
  pm2: '[PM2] Spawning PM2 daemon with pm2_home=/Users/xo/.pm2',
  gulp: '',
  grunt: '',
  git: '2.17.1',
  tsc: '2.9.2',
  mysql: '',
  redis: '4.0.10',
  mongodb: '',
  nginx: '',
  php: '5.6.30' }
```

### Steps to reproduce the problem

```js
await si.versions();
```

### Specifications 

- package version: 3.45.9
- operating system: macOS 10.12.6 (16G1510)
- hardware: Macbook Pro 2010

### Problem and possible solution

- `openssl` is including the `\n` along with a space.
- `openssl` is returning a different version to what I was expecting. `openssl version` shows `OpenSSL 1.0.2o  27 Mar 2018`.
- pm2 should show `2.7.2`.

**Comment by sebhildebrandt** [comment]
@OmgImAlexis - thank you for reporting that. Can you provide the output of the following commands:

```
pm2 -v
openssl version
```

One more thing concerting `openssl`: This (at the moment) refers to the version inside node.js - so basically `process.versions.openssl` but it would also be nice to see the installed version on the machine. 

I will implement a solution for that (no need for a PR)

**Comment by OmgImAlexis** [comment]
Thank you.

```
➜  api git:(master) ✗ pm2 -v
2.7.2

➜  api git:(master) ✗ openssl version
OpenSSL 1.0.2o  27 Mar 2018

➜  api git:(master) ✗ node
> process.versions.openssl
'1.1.0i \n'
```

**Comment by sebhildebrandt** [comment]
Thank you!! Very clear! Interesting where the `'[PM2] Spawning PM2 daemon with pm2_home=/Users/xo/.pm2'`comes from. Can you reproduce this PM2 output on the command line?

**Comment by OmgImAlexis** [comment]
No clue, I even tried manually this with `2.7.2` being returned. I've seen the "spawning" line show up about 5 times today while using the lib in my application.

```js
exec('pm2 -v', (error, stdout) => { console.log(stdout.toString()); });
```

**Comment by sebhildebrandt** [comment]
OK, no problem! I will try to reproduce this and will find a solution to avoid this output ;-)

**Comment by sebhildebrandt** [comment]
So I now added openssl version (and library) the OS is using (besides the one in nodejs).

Concerning the PM2 version: when launching a PM2 command and if the daemon wasn't started it will print `Spawning PM2 daemon`, so it means that your PM2 daemon was stopped. Sometimes this happens when the disk space of your server is full. Maybe its worth checking the logs at `~/.pm2/logs`

I implemented a fix that will at least not output this missleading version string.
